spynnaker/4.0.0/PyNNOnSpiNNakerExtensions.md

```diff
@@ -0,0 +1,21 @@
+---
+title: PyNN on SpiNNaker Extension Development Guide
+---
+
+This guide is intended to help you set up your development environment so that you can write new extensions to the implementation of SpiNNaker on PyNN.  This might include new neural models, or new connectors.  Note that this guide is not intended to help you make changes to the implementation itself, although it is conceivable that once your extension is stable, it might find its way in to the core code base.
+
+# sPyNNaker C Development Environment
+If you are writing new neural models, synapse models or plasticity models, it is likely that you will need to compile C code to run on SpiNNaker.  Firstly, you must [follow the instructions](/common_pages/3.0.0/CDevelopmentForSpiNNaker.html) to install a C development environment.  If your code is then to make use of the sPyNNaker neural modelling code base (which avoids the need to rewrite much of the necessary code):
+
+1. Download the sPyNNaker source code as a [zip](https://github.com/SpiNNakerManchester/sPyNNaker/archive/3.0.0.zip) or [tar.gz](https://github.com/SpiNNakerManchester/sPyNNaker/archive/3.0.0.tar.gz)
```

Comment:
I suspect these should refer to sPyNNaker 4.0.0 - though there probably isn't a zip file just now.  It could instead use "master.zip" and "master.tar.gz"